File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -0,0 +1,932 @@
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+Stream inputStream = assembly.GetManifestResourceStream("SampleBrowser.Presentation.Samples.Template.Sample.pptx");
+//Loads or open an PowerPoint Presentation
+IPresentation pptxDoc = Presentation.Open(inputStream);
+ISlide slide = pptxDoc.Slides[0];
+//Finds all the occurrences of a particular text in the specific slide
+ITextSelection[] textSelections = slide.FindAll("product", false, false);
+foreach (ITextSelection textSelection in textSelections)
+{
+	//Gets the found text as single text part
+	ITextPart textPart = textSelection.GetAsOneTextPart();
+	//Replace the text
+	textPart.Text = "Service";
+}
+//Create new memory stream to save Presentation.
+MemoryStream stream = new MemoryStream();
+//Save Presentation in stream format.
+pptxDoc.Save(stream);
+//Close the presentation
+pptxDoc.Close();
+stream.Position = 0;
+//The operation in Save under Xamarin varies between Windows Phone, Android and iOS platforms. Please refer presentation/xamarin section for respective code samples.
+if (Device.OS == TargetPlatform.WinPhone || Device.OS == TargetPlatform.Windows)
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and highlight in PowerPoint presentation
+
+.NET PowerPoint library allows you to find all the occurences of a text in the PowerPoint presentation and highlight it.
+
+The following code example illustrates how to find all the occurences of a particular text and highlight it.
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C#" %}
+//Opens an existing presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+	//Finds all the occurrences of a particular text in the PowerPoint presentation
+	ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+	foreach (ITextSelection textSelection in textSelections)
+	{
+		//Gets the found text containing text parts
+		foreach (ITextPart textPart in textSelection.GetTextParts())
+		{
+			//Sets highlight color
+			textPart.Font.HighlightColor = ColorObject.Yellow;
+		}
+	}
+	//Saves the Presentation	
+	pptxDoc.Save("Output.pptx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Opens an existing presentation.
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+	'Finds all the occurrences of a particular text in the PowerPoint presentation
+	Dim textSelections As ITextSelection() = pptxDoc.FindAll("product", False, False)
+	For Each textSelection As ITextSelection In textSelections
+		'Gets the found text containing text parts
```

Comment:
Gets the found text containing text parts

Reply:
Modified as you said.